enterprise_access/apps/subsidy_request/migrations/0004_update_subsidy_states.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 3.2.11 on 2022-02-07 18:25
+
+from django.db import migrations, models
+
+from enterprise_access.apps.subsidy_request.constants import SubsidyRequestStates
+
+def update_subsidy_request_states(apps, schema_editor):
+    LicenseRequest = apps.get_model('subsidy_request', 'LicenseRequest')
+    CouponCodeRequest = apps.get_model('subsidy_request', 'CouponCodeRequest')
+
+    new_states = {
+        'pending_review': SubsidyRequestStates.REQUESTED,
+        'approved_pending': SubsidyRequestStates.PENDING,
+        'approved_fulfilled': SubsidyRequestStates.APPROVED,
+        'denied': SubsidyRequestStates.DECLINED
+    }
+
+    for request in LicenseRequest.objects.all().iterator():
+        request.state = new_states.get(request.state)
+        request.save()
+
+    for request in CouponCodeRequest.objects.all().iterator():
+        request.state = new_states.get(request.state)
+        request.save()
```

Comment:
optional nit/consideration: Do you think it's worth being a bit more defensive around setting the `request.state`? For example, what is `new_states.get(request.state)` doesn't return anything because the current state isn't one of the previous options? I don't think this should happen but it might be worth considering _juuust_ in case; that way we won't unintentionally update existing requests if we can deduce its updated state 🤔

Reply:
Also, one other thought that just occurred to me... while the number of objects for requests are limited right now, do you think it'd be worth doing these operations as a `bulk_update` instead of calling `.save()` on each record?

Reply:
it should not happen, but it doesn't hurt to add request.state as a default so it doesn't get change.

Reply:
refactored!

Reply:
Nice! 🚀